app/uk/gov/hmrc/thirdpartyapplication/models/JsonFormatters.scala

```diff
@@ -271,3 +271,8 @@ object APIStatusJson {
   }
 
 }
+
+object ApplicationEventFormats {
+  implicit val actorFormats = Json.format[Actor]
+  implicit val teamMemberAddedEventFormats = Json.format[TeamMemberAddedEvent]
```

Comment:
I think we need to use the json union library and create a formatter for the ApplicationEvent trait instead

Reply:
ok will add this